lib/utilities/babel-config-for-targets.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+const presetEnv = require('babel-preset-env').default;
```

Comment:
The idea is that ember-cli-babel is "just" transpiling with the provided config options. I am not really opposed to flip flopping if folks feel strongly...